Sources/MapboxMaps/Foundation/Camera/CameraAnimator.swift

```diff
@@ -74,27 +123,69 @@ public class CameraAnimator: NSObject {
         propertyAnimator.finishAnimation(at: .current)
     }
 
-    /// Add animations block to the animator with a `delayFactor`.
-    public func addAnimations(_ animations: @escaping () -> Void, delayFactor: Double) {
-        // if this cameraAnimator is not in the list of CameraAnimators held by the `CameraManager` then add it to that list
-        propertyAnimator.addAnimations(animations, delayFactor: CGFloat(delayFactor))
-    }
-
     /// Add animations block to the animator.
-    public func addAnimations(_ animations: @escaping () -> Void) {
-        propertyAnimator.addAnimations(animations)
+    internal func addAnimations(_ animations: @escaping (inout CameraTransition) -> Void) {
+        animation = animations
     }
```

Comment:
this method seems unnecessary at this point

Reply:
I agree it's unnecessary at this time. But it does make it easier for when we do bring back the ability to add multiple animation blocks, which i suspect we'll need to do soon.